package.json

```diff
@@ -86,6 +86,21 @@
                         "onExP",
                         "preview"
                     ]
+                },
+                "python-envs.extensionPackageTrust": {
```

Comment:
This is not really trust, it is more of automatic package management. So this should likely say, `allowAutoPackageManagement`

/cc @cwebster-99 @luabud

Reply:
switched it to this for now but very flexible on this name - don't have a strong preference at the moment